lib/Parser/Parse.cpp

```diff
@@ -837,6 +837,14 @@ Symbol* Parser::AddDeclForPid(ParseNodePtr pnode, IdentPtr pid, SymbolType symbo
             }
             break;
         case knopVarDecl:
+
```

Comment:
Moved to below case knopVarDecl.
Treating function declaration like lexical declaration is unique to module top-level.